README.md

```diff
@@ -22,7 +22,6 @@ Make sure you have at least 1.5 GB free disk space.
 
     ```
     sudo adduser ff-node-monitor --home /opt/ff-node-monitor --system
-    cd /opt/ff-node-monitor
```

Comment:
Why did you move this?

Reply:
I had to

Reply:
So it is more obvious, that rust is installed one directory above the git repository

Reply:
And it belongs logically to the rust install step